src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1789,6 +1887,7 @@
 helps['network vnet peering create'] = """
     type: command
     short-summary: Create a peering.
+    long-summary: To learn more about VNet Peering about how to create a peering, visit https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-manage-peering#before-you-begin
```

Comment:
Remove `about how to create a peering`, change `visit` to `see`.

Reply:
Addressed.